docs/modules/ROOT/pages/whats-new.adoc

```diff
@@ -10,3 +10,5 @@ Below are the highlights of the release.
 * https://github.com/spring-projects/spring-security/pull/11771[gh-11771] - `HttpSecurityDsl` should support `apply` method
 * https://github.com/spring-projects/spring-security/pull/11232[gh-11232] - `ClientRegistrations#rest` defines 30s connect and read timeouts
 * https://github.com/spring-projects/spring-security/pull/11464[gh-11464] - Remember Me supports SHA256 algorithm
+* https://github.com/spring-projects/spring-security/issues/11347[gh-11347] - Support RequestMatcher Validation
```

Comment:
I think this has evolved. Rather than validation we have changed the DSL to make it easier to make the right choice.